crates/sdk/src/lib.rs

```diff
@@ -0,0 +1,49 @@
+//! Blueprint SDK
+
+#[cfg(feature = "tangle")]
+pub use tangle_deps::*;
+#[cfg(feature = "tangle")]
+mod tangle_deps {
+    pub use tangle_blueprint_sdk::*;
+}
+
+#[cfg(feature = "testing")]
+pub use testing::*;
+#[cfg(feature = "testing")]
+/// Testing utilities and helpers
+mod testing {
```

Comment:
```suggestion
#[cfg(feature = "testing")]
/// Testing utilities and helpers
pub mod testing {
```

Keep the module public, `use sdk::testing::utils` would be better than `use sdk::testing_utils` for example.

Reply:
Agreed